package.json

```diff
@@ -38,31 +38,31 @@
     "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "dependencies": {
```

Comment:
Are the dependencies and devDependencies changed motivated by the node 12 addition? In other words, if the modules are not upgraded this way node 12 job in travis fails?

Reply:
No, as summarized in the intro, they have been updated to the latest **supported** version for node 8 - this closes several outstanding vulnerabilities which were present due to using outdated version. 

I had assumed that the older versions were being kept for **node 6** support.

Reply:
Please add the changes to CHANGES_NEXT_RELEASE in this style:

```
- Upgrade from node-uuid ~1.4.1 to uuid ~3.3.2
```